File: Famoser.SyncApi/Api/Communication/Request/CollectionEntityRequest.cs
using System.Collections.Generic;
using Famoser.SyncApi.Api.Communication.Entities;
using Famoser.SyncApi.Api.Communication.Request.Base;
using Famoser.SyncApi.Enums;

namespace Famoser.SyncApi.Api.Communication.Request
{
    public class CollectionEntityRequest : BaseRequest
    {
        public List<CollectionEntity> CollectionEntities { get; set; } = new List<CollectionEntity>();
    }
}

File: Famoser.SyncApi.Webpage/src/Models/Communication/Request/CollectionEntityRequest.php
<?php
/**
 * Created by PhpStorm.
 * User: famoser
 * Date: 04.11.2016
 * Time: 19:15
 */

namespace Famoser\SyncApi\Models\Communication\Request;


use Famoser\SyncApi\Models\Communication\Entities\CollectionEntity;
use Famoser\SyncApi\Models\Communication\Request\Base\BaseRequest;

class CollectionEntityRequest extends BaseRequest
{
    /* @var CollectionEntity[] $CollectionEntities */
    public $CollectionEntities;
}